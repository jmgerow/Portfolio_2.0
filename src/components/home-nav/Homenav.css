.home-nav-item {
    text-decoration: none;
    color: #546e7a;
    transition: transform .2s;
} 

.home-nav-item:hover {
    transform: scale(1.1);
}

.home-nav-icon {
    text-decoration: none;
    color: #546e7a;
} 