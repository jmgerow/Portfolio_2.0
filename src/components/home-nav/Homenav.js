import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import './Homenav.css';

class Homenav extends Component {

    render() {
        return (
            <div className="home-nav container">
                <div className="row center blue-grey-text text-darken-1" id="navigation">
                    <div className="col s12 m6 xl3 home-nav-item">
                        <Link to="/portfolio" className="home-nav-icon">
                            <h3>Portfolio</h3>
                            <i className="fas fa-laptop fa-6x"></i>
                        </Link>
                    </div>
                    <div className="col s12 m6 xl3 home-nav-item">
                        <Link to="/about" className='home-nav-icon' >
                            <h3>Bio</h3>
                            <i className="far fa-user fa-6x"></i>
                        </Link>
                    </div>
                    <div className="col s12 m6 xl3 home-nav-item">
                        <Link to="/resume" className='home-nav-icon'>
                            <h3>Resume</h3>
                            <i className="far fa-file-alt fa-6x"></i>
                        </Link>
                    </div>
                    <div className="col s12 m6 xl3 home-nav-item">
                        <Link to="contact" className='home-nav-icon'>
                            <h3>Contact</h3>
                            <i className="far fa-comment fa-6x"></i>
                        </Link>
                    </div>
                </div>
            </div>
        );
    }
}

export default Homenav;