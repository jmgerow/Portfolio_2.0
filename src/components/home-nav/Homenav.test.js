import React from 'react';
import { shallow } from 'enzyme';
import Homenav from './Homenav';

describe('Homenav', () => {
    const homenav = shallow(<Homenav />)
  
    it('renders without crashing', () => {
      expect(homenav).toMatchSnapshot();
    });
 
    // it('displays name', () => {
    //   expect(navbar.state().title).toEqual("Justin Gerow")
    // });
  });