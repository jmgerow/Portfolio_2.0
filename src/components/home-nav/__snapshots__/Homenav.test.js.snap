// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homenav renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Homenav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="row center blue-grey-text text-darken-1"
        id="navigation"
      >
        <div
          className="col s12 m6 xl3 home-nav-item"
        >
          <Link
            className="home-nav-icon"
            to="/portfolio"
          >
            <h3>
              Portfolio
            </h3>
            <i
              className="fas fa-laptop fa-6x"
            />
          </Link>
        </div>
        <div
          className="col s12 m6 xl3 home-nav-item"
        >
          <Link
            className="home-nav-icon"
            to="/about"
          >
            <h3>
              About Me
            </h3>
            <i
              className="far fa-user fa-6x"
            />
          </Link>
        </div>
        <div
          className="col s12 m6 xl3 home-nav-item"
        >
          <Link
            className="home-nav-icon"
            to="/resume"
          >
            <h3>
              Resume
            </h3>
            <i
              className="far fa-file-alt fa-6x"
            />
          </Link>
        </div>
        <div
          className="col s12 m6 xl3 home-nav-item"
        >
          <Link
            className="home-nav-icon"
            to="contact"
          >
            <h3>
              Contact
            </h3>
            <i
              className="far fa-comment fa-6x"
            />
          </Link>
        </div>
      </div>,
      "className": "home-nav container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="/portfolio"
            >
              <h3>
                Portfolio
              </h3>
              <i
                className="fas fa-laptop fa-6x"
              />
            </Link>
          </div>,
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="/about"
            >
              <h3>
                About Me
              </h3>
              <i
                className="far fa-user fa-6x"
              />
            </Link>
          </div>,
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="/resume"
            >
              <h3>
                Resume
              </h3>
              <i
                className="far fa-file-alt fa-6x"
              />
            </Link>
          </div>,
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="contact"
            >
              <h3>
                Contact
              </h3>
              <i
                className="far fa-comment fa-6x"
              />
            </Link>
          </div>,
        ],
        "className": "row center blue-grey-text text-darken-1",
        "id": "navigation",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="home-nav-icon"
              to="/portfolio"
            >
              <h3>
                Portfolio
              </h3>
              <i
                className="fas fa-laptop fa-6x"
              />
            </Link>,
            "className": "col s12 m6 xl3 home-nav-item",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  Portfolio
                </h3>,
                <i
                  className="fas fa-laptop fa-6x"
                />,
              ],
              "className": "home-nav-icon",
              "to": "/portfolio",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Portfolio",
                },
                "ref": null,
                "rendered": "Portfolio",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fas fa-laptop fa-6x",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="home-nav-icon"
              to="/about"
            >
              <h3>
                About Me
              </h3>
              <i
                className="far fa-user fa-6x"
              />
            </Link>,
            "className": "col s12 m6 xl3 home-nav-item",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  About Me
                </h3>,
                <i
                  className="far fa-user fa-6x"
                />,
              ],
              "className": "home-nav-icon",
              "to": "/about",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "About Me",
                },
                "ref": null,
                "rendered": "About Me",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "far fa-user fa-6x",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="home-nav-icon"
              to="/resume"
            >
              <h3>
                Resume
              </h3>
              <i
                className="far fa-file-alt fa-6x"
              />
            </Link>,
            "className": "col s12 m6 xl3 home-nav-item",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  Resume
                </h3>,
                <i
                  className="far fa-file-alt fa-6x"
                />,
              ],
              "className": "home-nav-icon",
              "to": "/resume",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Resume",
                },
                "ref": null,
                "rendered": "Resume",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "far fa-file-alt fa-6x",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="home-nav-icon"
              to="contact"
            >
              <h3>
                Contact
              </h3>
              <i
                className="far fa-comment fa-6x"
              />
            </Link>,
            "className": "col s12 m6 xl3 home-nav-item",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <h3>
                  Contact
                </h3>,
                <i
                  className="far fa-comment fa-6x"
                />,
              ],
              "className": "home-nav-icon",
              "to": "contact",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Contact",
                },
                "ref": null,
                "rendered": "Contact",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "far fa-comment fa-6x",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row center blue-grey-text text-darken-1"
          id="navigation"
        >
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="/portfolio"
            >
              <h3>
                Portfolio
              </h3>
              <i
                className="fas fa-laptop fa-6x"
              />
            </Link>
          </div>
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="/about"
            >
              <h3>
                About Me
              </h3>
              <i
                className="far fa-user fa-6x"
              />
            </Link>
          </div>
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="/resume"
            >
              <h3>
                Resume
              </h3>
              <i
                className="far fa-file-alt fa-6x"
              />
            </Link>
          </div>
          <div
            className="col s12 m6 xl3 home-nav-item"
          >
            <Link
              className="home-nav-icon"
              to="contact"
            >
              <h3>
                Contact
              </h3>
              <i
                className="far fa-comment fa-6x"
              />
            </Link>
          </div>
        </div>,
        "className": "home-nav container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="col s12 m6 xl3 home-nav-item"
            >
              <Link
                className="home-nav-icon"
                to="/portfolio"
              >
                <h3>
                  Portfolio
                </h3>
                <i
                  className="fas fa-laptop fa-6x"
                />
              </Link>
            </div>,
            <div
              className="col s12 m6 xl3 home-nav-item"
            >
              <Link
                className="home-nav-icon"
                to="/about"
              >
                <h3>
                  About Me
                </h3>
                <i
                  className="far fa-user fa-6x"
                />
              </Link>
            </div>,
            <div
              className="col s12 m6 xl3 home-nav-item"
            >
              <Link
                className="home-nav-icon"
                to="/resume"
              >
                <h3>
                  Resume
                </h3>
                <i
                  className="far fa-file-alt fa-6x"
                />
              </Link>
            </div>,
            <div
              className="col s12 m6 xl3 home-nav-item"
            >
              <Link
                className="home-nav-icon"
                to="contact"
              >
                <h3>
                  Contact
                </h3>
                <i
                  className="far fa-comment fa-6x"
                />
              </Link>
            </div>,
          ],
          "className": "row center blue-grey-text text-darken-1",
          "id": "navigation",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="home-nav-icon"
                to="/portfolio"
              >
                <h3>
                  Portfolio
                </h3>
                <i
                  className="fas fa-laptop fa-6x"
                />
              </Link>,
              "className": "col s12 m6 xl3 home-nav-item",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    Portfolio
                  </h3>,
                  <i
                    className="fas fa-laptop fa-6x"
                  />,
                ],
                "className": "home-nav-icon",
                "to": "/portfolio",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Portfolio",
                  },
                  "ref": null,
                  "rendered": "Portfolio",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fas fa-laptop fa-6x",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
              ],
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="home-nav-icon"
                to="/about"
              >
                <h3>
                  About Me
                </h3>
                <i
                  className="far fa-user fa-6x"
                />
              </Link>,
              "className": "col s12 m6 xl3 home-nav-item",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    About Me
                  </h3>,
                  <i
                    className="far fa-user fa-6x"
                  />,
                ],
                "className": "home-nav-icon",
                "to": "/about",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "About Me",
                  },
                  "ref": null,
                  "rendered": "About Me",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "far fa-user fa-6x",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
              ],
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="home-nav-icon"
                to="/resume"
              >
                <h3>
                  Resume
                </h3>
                <i
                  className="far fa-file-alt fa-6x"
                />
              </Link>,
              "className": "col s12 m6 xl3 home-nav-item",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    Resume
                  </h3>,
                  <i
                    className="far fa-file-alt fa-6x"
                  />,
                ],
                "className": "home-nav-icon",
                "to": "/resume",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Resume",
                  },
                  "ref": null,
                  "rendered": "Resume",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "far fa-file-alt fa-6x",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
              ],
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Link
                className="home-nav-icon"
                to="contact"
              >
                <h3>
                  Contact
                </h3>
                <i
                  className="far fa-comment fa-6x"
                />
              </Link>,
              "className": "col s12 m6 xl3 home-nav-item",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <h3>
                    Contact
                  </h3>,
                  <i
                    className="far fa-comment fa-6x"
                  />,
                ],
                "className": "home-nav-icon",
                "to": "contact",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Contact",
                  },
                  "ref": null,
                  "rendered": "Contact",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "far fa-comment fa-6x",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
              ],
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
